Doc/library/datetime.rst

```diff
@@ -2601,8 +2601,8 @@ Notes:
 
 (9)
    When used with the :meth:`strptime` method, the leading zero is optional
-   for  formats ``%d``, ``%m``, ``%H``, ``%I``, ``%M``, ``%S``, ``%J``, ``%U``,
```

Comment:
It looks correct, as lowercase matches the [table](https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes), but I'm less sure about the alphabetical change. The current ordering matches the table's so it might be better to keep that unchanged as there's some logical reasoning to it, but I'll defer the judgement here.

Reply:
Good call on the inconsistency, @slateny. The ordering is now changed back to the original.